src/modules/translation/languages/es.json

```diff
@@ -134,11 +134,12 @@
   },
   "claim_ens_page": {
     "title": "Reclamar un nombre",
-    "subtitle": "Una asignación de nombre permite tener una URL unica para tu parcela con el siguiente formato: http://name.dcl.eth.link. {separator} Cuando los usuarios accedan a http://name.dcl.eth.link, serán redirijidos a tu parcela en el {dclWorldLink}.", 
-    "world": "Mundo Decentraland",
+    "subtitle": "Una asignación de nombre le permite tener una URL única para su terreno con el siguiente formato: http://name.dcl.eth.link. Una vez que los usuarios abren el enlace, serán redirigidos a él en {dclWorldLink}", 
+    "world": "Mundo Decentraland!",
```

Comment:
I guess that we can use just `Decentraland`. `Mundo Decentraland` does not make any sense.